apps/anoma_node/lib/examples/esolver.ex

```diff
@@ -113,20 +115,15 @@ defmodule Anoma.Node.Examples.ESolver do
 
     tx = Examples.ETransparent.ETransaction.swap_from_actions()
 
-    IntentPool.new_intent(node_id, tx)
-
-    tx_candidate = [
-      [1, 0, [1 | tx |> Noun.Nounable.to_noun()], 0 | 909],
-      0 | 707
-    ]
-
     tx_filter = [Anoma.Node.Event.node_filter(node_id), %Mempool.TxFilter{}]
 
     with_subscription [tx_filter] do
-      Mempool.tx(
-        node_id,
-        {:transparent_resource, tx_candidate}
-      )
```

Comment:
I dropped it since it seems to be redundant. From what I understand the Solver is supposed to call it. Does it make sense?

Reply:
Looks good to me!

Thanks once again for all your help!